components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineView.kt

```diff
@@ -237,6 +249,26 @@ class SystemEngineView @JvmOverloads constructor(
         return false
     }
 
+    internal fun addFullScreenView(view: View, callback: WebChromeClient.CustomViewCallback) {
+        val webView = findViewWithTag<WebView>("webview")
+        webView?.apply { this.visibility = View.GONE }
+
+        fullScreenCallback = callback
+
+        view.tag = "fullscreen"
```

Comment:
nit: We should make sure that we do not clash with tags/ids of the consuming apps. For most resources we use a "mozac" prefix and sometimes another "module" prefix (e.g. Strings: `mozac_browser_toolbar_menu_button)`.

Reply:
I used `mosac_system_engine_*`.